src/core/open-file.c

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <errno.h>
+#include <fcntl.h>
+#include <string.h>
+
+#include "open-file.h"
+#include "string-util.h"
+#include "strv.h"
+
+int parse_open_file(const char *v, OpenFile *ret) {
```

Comment:
Given that we can't use this function in the D-Bus logic, it should be inlined into `config_parse_open_file()`